package.json

```diff
@@ -100,8 +100,8 @@
     "jest-jasmine-ci": "yarn jest-jasmine --color --config jest.config.ci.js",
     "jest-coverage": "yarn jest --coverage",
     "lint": "eslint . --cache --ext js,jsx,ts,tsx,md",
-    "lint:prettier": "prettier '**/*.{md,yml,yaml}' 'website/**/*.{css,js}' --write --ignore-path .gitignore",
-    "lint:prettier:ci": "prettier '**/*.{md,yml,yaml}' 'website/**/*.{css,js}' --check --ignore-path .gitignore",
```

Comment:
without the ignore file thing we should make sure to ignore`build` output in `.prettierignore`.

https://github.com/facebook/jest/blob/b4de6cb6e75532447d62115f8cbcc62abcf10c7a/.gitignore#L17-L22
https://github.com/facebook/jest/blob/b4de6cb6e75532447d62115f8cbcc62abcf10c7a/.gitignore#L35-L53